### DISCUSSION TITLE
Richiesta credenziali

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#215] [Idee e proposte]
Possibilità ai genitori di iscrivere il figlio a scuola attraverso un form... la segreteria accetterà l'iscrizione e verranno generate le credenziali autonomamente senza inserire alcun dato

**Comment by sempliciniluigi89** [comment]
Eccellente idea

Il Lun 21 Mar 2022, 18:03 Manu ***@***.***> ha scritto:
